lib/jxl/blending.cc

```diff
@@ -36,7 +36,8 @@ Status ImageBlender::PrepareBlending(
     const std::vector<ExtraChannelInfo>* extra_channel_info,
     const ColorEncoding& frame_color_encoding, const Rect& frame_rect,
     Image3F* output, const Rect& output_rect,
-    std::vector<std::pair<ImageF*, Rect>> output_extra_channels) {
+    std::vector<ImageF>* output_extra_channels,
+    std::vector<Rect> output_extra_channels_rects) {
```

Comment:
wouldn't this be a `std::vector<Rect>&&`? or it doesn't matter because you call it with std::move() anyway?

Reply:
Yep, the latter. https://abseil.io/tips/117